notebooks/vertex_genai/solutions/gen_ai_evaluation_service.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>Some metrics like "fluency", "coherence", "safety", "groundedness" are not computation-based but model-based with pre-defined prompt?</p><p> </p><p>Good to remove if it's confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Thanks, removed the non-computation based metrics.</p>